packages/min/src/base-component.ts

```diff
@@ -38,6 +38,24 @@ export class BaseComponent<T extends HTMLElement = HTMLElement> extends Control<
       this.append(child);
     }
   }
+
+  public stylize<K extends keyof CSSStyleDeclaration>(style: K, value: CSSStyleDeclaration[K]): void;
+  public stylize(style: Partial<CSSStyleDeclaration>): void;
+  public stylize<K extends keyof CSSStyleDeclaration>(
+    style: K | Partial<CSSStyleDeclaration>,
+    value?: CSSStyleDeclaration[K],
+  ): void {
+    if (typeof style !== 'object' && value) {
+      this._node.style[style] = value;
+      return;
+    }
+
+    Object.assign(this._node.style, style);
+  }
+
+  public unstylize<K extends keyof CSSStyleDeclaration>(key: K) {
```

Comment:
Maybe overload this method too. Like you did above in the method `stylize()`? And change the name of it to *style*?